actionview/blade.yml

```diff
@@ -3,9 +3,3 @@ load_paths:
 
 logical_paths:
   - rails-ujs.js
```

Comment:
Do we still new this entire config?

Reply:
The `load_paths` is still required as blade is still being used to run the ujs tests. 

The tests can run without the `logical_paths` being set - but can error if the `log` directory exists. Having this set ensures that the tests can run even if the file system is in a bad state - so it is probably better to leave it in here for now.

I intend to follow up with looking at how we can replace blade for running the tests - but that doesn't seem to be required to address the original issue at https://github.com/rails/rails/issues/45407.